### PR TITLE
dropzone-target-option CSH-5578: add data-dropzone-target option to a…

### PR DESCRIPTION
**Pull request by wwsno**
…uthor component template. Update documentation describing this option, as well as doc-image directive in general.

### CODE REVIEW

File: docs/directives/IMAGE.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# `doc-image`
+
+Binds an image to a html element. For `IMG` tag elements the image url is bound to the `src` attribute:
```
Comment (wwjhu) [inline docs/directives/IMAGE.md]: What does this mean? If you want to use `doc-image` you need to set the `src` attribute? Or does that it mean the editor will set the `src` attribute for you?

File: docs/directives/IMAGE.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    </div>
+
+The value of this attribute must match the directive key.
+
+For example usage see the `author` component in the boilerplate project.
+
+## Hiding droppable placeholder
+A placeholder overlay is shown when hovering the mouse of the droppable target element of the directive, informing the user an image can be dropped and what type of images are accepted.
+
+For some images this droppable placeholder is too big to display properly inside the editor. You can hide this placeholder by using the attribute `data-placeholder-style`. Currently only accepts the value `hidden`:
```
Comment (wwjhu) [inline docs/directives/IMAGE.md]: Should it be added to the `data-drop-target` element when you have that or always the `doc-image` element? Maybe good to add.

File: docs/directives/IMAGE.md (hunk 1 above)
Comment (wwsno) [reply docs/directives/IMAGE.md]: The editor sets it for the user. I just wanted to describe the behavior of the image directive. Should I leave it out, or keep it?

File: docs/directives/IMAGE.md (hunk 2 above)
Comment (wwsno) [reply docs/directives/IMAGE.md]: Updated.

**Pull request merged**